README.md

```diff
@@ -59,7 +59,7 @@ $ mvn org.commonjava.maven.plugins:directory-maven-plugin:highest-basedir@resolv
 Build with the `dist` profile as shown above, then execute this:
 
 ```
-$ java -jar kroxylicious/target/kproxy-1.0-SNAPSHOT.jar
+$ java -jar kroxylicious/target/kroxylicious-1.0-SNAPSHOT.jar
```

Comment:
The name seemed to be different for me compared to the README, so I'm updating it.
I cannot actually run this though, as I get this error:
`java -jar kroxylicious/target/kroxylicious-1.0-SNAPSHOT.jar 
Missing required option: '--config=<configFile>'`
What config file is this? And how do I create one, so I can add this step in here also?